packages/tocco-ui/src/EditableValue/typeEditors/MultiRemoteSelect.js

```diff
@@ -79,14 +79,12 @@ class MultiRemoteSelect extends React.Component {
 
 MultiRemoteSelect.propTypes = {
   onChange: PropTypes.func,
-  value: PropTypes.arrayOf(
-    PropTypes.shape({
-      key: PropTypes.oneOfType([
-        PropTypes.string,
-        PropTypes.number
-      ])
-    })
-  ),
+  value: PropTypes.shape({
```

Comment:
Why did you remove array type? Its a **multi** remote field so its must be an array